yarn.lock

```diff
@@ -16,6 +16,16 @@
     esutils "^2.0.2"
     js-tokens "^4.0.0"
 
```

Comment:
this should go away after you remove circular dependency and run `yarn`

Reply:
They did 👍